.travis.yml

```diff
@@ -1,11 +0,0 @@
-language: ruby
```

Comment:
Is there a reason we still have travis setup with GitHub Actions?

Reply:
Only legacy. Since GitHub Actions is stable now, it’s safe to sunset Travis.